cmake/projects/Android-SDK-Tools/hunter.cmake

```diff
@@ -62,6 +62,17 @@ elseif(CMAKE_HOST_UNIX)
       PACKAGE_NAME
       Android-SDK-Tools
       VERSION
+      "24.3"
+      URL
+      "http://dl-ssl.google.com/android/repository/tools_r24.3-linux.zip"
+      SHA1
+      ecb86253e97cd14459888701fc3478b4d153a86c
+  )
+
+  hunter_add_version(
+      PACKAGE_NAME
+      Android-SDK-Tools
+      VERSION
```

Comment:
Yeah, I can't really explain how this got there, I used the "squashing" instructions from last time to generate the commit, somehow this pushed through...